content/guides/component-testing/component-framework-configuration.md

```diff
@@ -554,6 +554,52 @@ export default defineConfig({
 </template>
 </cypress-config-file>
 
+#### Options API
```

Comment:
Is this meant to be nested under `Angular CLI`, or should it just be under `Angular`?